templates/startup_script.sh.tmpl

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# create tamr user, to run tamr
+adduser --shell /bin/bash --home /home/tamr tamr
+
+# make tamr_home_directory directory if it doesn't already exist
+sudo /bin/bash -c 'mkdir -p ${tamr_home_directory} ; chown -R tamr:tamr ${tamr_home_directory}'
+
+
+
+TAMR_CONFIG=$(cat <<EOF
+${tamr_config}
+EOF
+)
+
+
+
+# check if tamr has been installed on the VM, if not download and install it
+if [[ ! -f ${tamr_home_directory}/tamr/start-unify.sh ]]; then
+    # download tamr install
+    sudo -S -u tamr -i /bin/bash -c 'aws s3 cp ${tamr_zip_uri} ${tamr_home_directory}/unify.zip'
+    # unzip and install unify
+    sudo -S -u tamr -i /bin/bash -c 'unzip ${tamr_home_directory}/unify.zip -d ${tamr_home_directory}'
```

Comment:
minor note that unzip is not installed by default on some linux systems but I think its fine assume it will be instead of sorting out a script to install it.

Reply:
Yes, the expectation is if any required packages unzip, postgres, etc are not installed then the variable bootstrap_scripts variable should be used to pass a script that uses those